### PR TITLE
[deprecation round2] LPS-104241 portal-kernel part 5

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian After reviewing the usages we decide to exclude some of the changes in the last pull (all search related). Some of them have complicated logic and needs in-depth check before removing; others don't seem to be appropriate (or, not something we can decide) for removing because they explicitly added logic for backwards compatibility of deprecated features.

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test

**Comment by dantewang** [comment]
ci:test:sf

**Comment by tinatian** [comment]
See https://github.com/shuyangzhou/liferay-portal/pull/8574

File: modules/apps/social/social-bookmarks-taglib/src/main/resources/META-INF/resources/bookmark/deprecated_bookmark.jsp [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <%@ include file="/bookmark/init.jsp" %>
 
 <%
-String icon = PropsUtil.get(PropsKeys.SOCIAL_BOOKMARK_ICON, new Filter(type));
+String icon = PropsUtil.get("social.bookmark.icon", new Filter(type));
```
Comment (tinatian) [inline modules/apps/social/social-bookmarks-taglib/src/main/resources/META-INF/resources/bookmark/deprecated_bookmark.jsp:20]: @dantewang @yuhai 
Please note I did not include this commit. Reason is this is not a real usage removal, actually it is a property inline, so let's keep it for now.

Tina.

**Pull request closed without merging**